src/examples/v3/exampleTransaction.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://trust.propdata.org.uk/schemas/v3/pdtf-transaction.json",
-  "transactionId": "3e4666bf-d5e5-4aa7-b8ce-cefe41c7568a",
+  "transactionId": "LnRdGiBUkLbnuNJ89p4CEj",
```

Comment:
Are we moving away from UUID4 for this?

Reply:
@jacknely sorry there's a mix in the examples but I seem to think we'd all agreed we'd use the base58 encoded versions of the UUIDs to make them shorter and more URL-friendly?